Adding support to override CloudEvents metadata when publishing an event (#130)
gsoft-inc/wl-domain-event-propagation: README.md

await this._eventPropagationClient.PublishDomainEventAsync(domainEvent, CancellationToken.None);
```

> It is possible to specify additional metadata on your domain events to allow leveraging the subscription filtering within Azure.

Reviewer: Simpler version, if you want it

```suggestion
You can specify additional metadata for your domain events to leverage subscription filtering within Azure.
```
